readme.md

```diff
@@ -553,6 +553,7 @@ The `options` object has the following shape:
     vendorPrefixes: boolean,
     isGlobal: boolean,
     filename: ?string, // defined only when styled-jsx/babel is used via Babel CLI,
+    sourceFileName: string, // the original Javascript file the CSS block was in
```

Comment:
this is also defined _when styled-jsx/babel is used via Babel CLI_.
i.e. if you use `transform` from `babel-core` and pass it the file content it won't have any of the `file` information.

Reply:
basically you need to prefix `string` with a `?` and copy the comment above.

Reply:
Hmm when I was working on this I was testing it by editing the package in my node_modules of a nextjs project, fileInfo.sourceFileName seemed to be present but fileInfo.filename wasn't, I'll check the Babel docs to be sure